### ISSUE TITLE
Crashing "Error:  Unhandled "error" event. ([object Object])

### ISSUE THREAD

**Issue by PixelBurstHD** [SM-Wistful/Discord-RoleReact#5]
The bot works fine most of the time, but I'm finding it crashing occasionally with the following output 
![image](https://user-images.githubusercontent.com/10819615/45649815-5bb8e380-bac4-11e8-80b3-248fbdf5dd6b.png)

Apologies for the printscreen as this was in a tmux session with other windows active and I couldn't get it to copy paste nicely!

**Comment by ipadpuppydogdude1** [comment]
This is not a issue with the code, it is your internet connection or your PC.

**Comment by PixelBurstHD** [comment]
> This is not a issue with the code, it is your internet connection or your PC.
> 
> **How did I find it this way?**
> 
> Because I had that error before and it was my internet or my PC.

You are right - however the error event not being handled is what leads to the crash.

A simple `bot.on('error', console.error);` fixes this, causing it to log the connection error instead of exiting the process, and when the connection returns it will continue working!

I'll submit a PR when I've got a free sec, but you can just copy that line into RoleReact.js for the time being.
